include/zephyr/linker/common-ram.ld

```diff
@@ -3,10 +3,11 @@
 #if defined(CONFIG_NETWORKING)
 #ifndef NETWORK_RAM_SECTIONS
 #define NETWORK_RAM_SECTIONS \
-	ITERABLE_SECTION_RAM(net_if, 4) \
-	ITERABLE_SECTION_RAM(net_if_dev, 4) \
-	ITERABLE_SECTION_RAM(net_l2, 4) \
-	ITERABLE_SECTION_RAM(eth_bridge, 4)
+	ITERABLE_SECTION_RAM(net_if,		4) \
+	ITERABLE_SECTION_RAM(net_if_dev,	4) \
+	ITERABLE_SECTION_RAM(net_l2,		4) \
+	ITERABLE_SECTION_RAM(conn_mgr_bound,	4) \
```

Comment:
likely this should also have its own cmake file

Reply:
We need to decide on the final location for the `conn_mgr`. However, from my perspective, while not being part of the core networking stack, it still falls under networking area, so IMO the current section location is acceptable.

Reply:
Agreed that this location is acceptable, (though being under `/lib/` is a bit odd). 

I wonder if it should live in `subsys/net/conn_mgr`?

Reply:
AFAIR it was one of the options considered, and for me it's just fine.